torch/_inductor/config.py

```diff
@@ -1501,6 +1501,8 @@ class test_configs:
     autotune_choice_name_regex: Optional[str] = None
     autotune_choice_desc_regex: Optional[str] = None
 
+    graphsafe_rng_func_ignores_fallback_random = False
```

Comment:
what use case do you imagine for this config? (I imagine most of the time, people either want to buy into all PT2 features whole-sale, or fall back 100% to eager randomness for reproducibility, but maybe I'm wrong). Also - if we keep this config, might be better to keep it next to the other rng confi?

Reply:
it's in the `test_configs` set for what its worth. I had to add it to test the inductor w/cudagraphs and aot eager above equality above. This is because:
- we need to apply fallback_random to get equal rng values for aot_eager and inductor
- but, I don't want to skip the cudagraphs safe rng bc that is point of the test.

I think i could work around this with using an rng op we dont decompose, like randperm

Reply:
we still need it even with randperm since we will fallback to non cudagraph rng to match numerics, then we cant test it..